### PR TITLE
used pictures from 2014 for site

### PR DESCRIPTION
**Pull request by Iepoev**

### CODE REVIEW

File: site/stylesheets/flexslider.css [hunk 1]
```diff
@@ -40,7 +40,7 @@ html[xmlns] .slides {display: block;}
 /* FlexSlider Default Theme
 *********************************/
 .flexslider {margin: 0 0 60px; background: #fff; border: 0; position: relative; -webkit-border-radius: 0; -moz-border-radius: 0; -o-border-radius: 0; border-radius: 0; box-shadow: 0 1px 4px rgba(0,0,0,.2); -webkit-box-shadow: 0 1px 4px rgba(0,0,0,.2); -moz-box-shadow: 0 1px 4px rgba(0,0,0,.2); -o-box-shadow: 0 1px 4px rgba(0,0,0,.2); zoom: 1;}
-.flex-viewport {max-height: 444px; -webkit-transition: all 1s ease; -moz-transition: all 1s ease; transition: all 1s ease;}
+.flex-viewport {max-height: 630px; -webkit-transition: all 1s ease; -moz-transition: all 1s ease; transition: all 1s ease;}
```
Comment (Iepoev) [inline site/stylesheets/flexslider.css]: old max-height cuts of the bottom of every picture. this max-height makes the viewport have a 10px empty ribbon on lower resolutions.

**Comment by Procrat** [comment]
site/images/animatie/fakbars.jpg is a bit shaken. I suggest another photograph.

**Comment by Procrat** [comment]
I think the max-height is set that way so as not to let the picture occupy the whole screen.

**Comment by Procrat** [comment]
As the picture are stretched to a width of 960, I think it's best to use the photos with the highest available resolution (960x637) instead of downscaled ones (720x477). Or was there a specific reason to do it like this?

**Comment by Iepoev** [comment]
I prefer to show the entire picture and taking up more space than simply cutting off the bottom 30% of the picture

720x477 is the native resolution of the facebook images (maybe theres some obscured way to get a higher resolution?).

**Comment by Procrat** [comment]
They're rather ambience photo's than a full-fledged album (like on Facebook). Not that it matters that much, but I don't really see the benefit.

The images on Facebook are all 960px in it's largest side. I don't know how you got the 720x477 ones.

**Comment by Procrat** [comment]
Looks nice ^^

**Pull request merged**